keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2062,6 +2300,17 @@ exist at all must not be allowed to run. Instead, a suitable event must be
 emitted which explains the problem. Such a situation can occur as part of
 downgrade scenarios.
 
+```
+<<[UNRESOLVED @pohly ]>>
+Should this use a dynamic client and unstructured.Unstructured to extract the
+relevant fields? That would make it possible to completely remove support
+for v1alpha2 when introducing v1beta1, as long as the field names and semantic
+remain the same.
+
+The alternative is to keep supporting older revisions for the sake of the kubelet.
+<<[/UNRESOLVED]>>
```

Comment:
I think it's fine if kubelet uses a certain version of the API for its own purposes. We can ensure that the apiserver provides a version of the API that is needed by an older kubelet as far back as we want to support version skew.

For example, suppose we switch to v1alpha3 in 1.31, a breaking change. Then in 1.32, we introduce v1beta1. Normally we would remove v1alpha3, but to try out version skew support, we can keep v1alpha3. Ideally this won't be hard as we don't plan any big changes for the API in 1.32. Then kubelet from v1.31 can be combined with drivers and control plane from v1.32 where both are using v1beta1.